packages/destination-actions/src/destinations/ironclad/index.ts

```diff
@@ -0,0 +1,46 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+
+import viewContract from './viewContract'
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Ironclad',
+  slug: 'actions-ironclad',
+  mode: 'cloud',
+
+  authentication: {
+    scheme: 'custom',
+    fields: {
+      sid: {
+        label: 'Site Access ID',
+        description:
+          'Site Access ID. An ID that’s unique for each Site within your account. Information on finding your sid can be found in the Authentication section',
```

Comment:
Sentences should end with a `.`. Make sure to regenerate the types so those files get updated as well.
```suggestion
          'Site Access ID. An ID that’s unique for each site within your account. Information on finding your sid can be found in the authentication section.',
```

Reply:
I think this supports markdown as well in case you want to link to the authentication section.

Reply:
Hi @marinhero, the Authentication Section lives within the Ironclad portal, maybe I could place a Screenshot link to this?